presto-native-execution/src/test/java/com/facebook/presto/nativeworker/ContainerQueryRunnerUtils.java

```diff
@@ -0,0 +1,284 @@
+        }
+        catch (IOException e) {
+            fail("Presto CLI failed to create log.properties and was exited with error message: " + e.getMessage());
+        }
+        return null;
+    }
+
+    public static Path createCoordinatorNodeProperties()
+    {
+        try {
+            Path tempConfigProperties = Files.createTempFile("node", ".properties");
+
+            String properties = "node.environment=testing\n" +
+                    "node.location=testing-location\n" +
+                    "node.data-dir=/var/lib/presto/data\n";
+            Files.write(tempConfigProperties, properties.getBytes(), StandardOpenOption.APPEND);
+
+            return tempConfigProperties;
+        }
+        catch (IOException e) {
+            fail("Presto CLI failed to create config.properties and was exited with error message: " + e.getMessage());
+        }
+        return null;
+    }
+
+    public static Path createNativeWorkerEntryPointScript()
+            throws IOException
+    {
+        Path entryPointScript = Files.createTempFile("entrypoint", ".sh");
+
+        String scriptContent =
+                "#!/bin/sh\n\n" +
+                        "GLOG_logtostderr=1 presto_server \\\n" +
+                        "    --etc-dir=/opt/presto-server/etc\n";
+        Files.write(entryPointScript, scriptContent.getBytes(), StandardOpenOption.WRITE);
+
+        entryPointScript.toFile().setExecutable(true);
+        return entryPointScript;
+    }
+
+    public static Path createCoordinatorEntryPointScript()
+    {
+        try {
+            Path entryPointScript = Files.createTempFile("entrypoint", ".sh");
+
+            String scriptContent =
+                    "#!/bin/sh\n" +
+                            "set -e\n" +
+                            "$PRESTO_HOME/bin/launcher run\n";
+            Files.write(entryPointScript, scriptContent.getBytes(), StandardOpenOption.WRITE);
+
+            entryPointScript.toFile().setExecutable(true);
+            return entryPointScript;
+        }
+        catch (IOException e) {
+            fail("Presto CLI failed to create entrypoint.sh and was exited with error message: " + e.getMessage());
+        }
+        return null;
+    }
+
+    static Path createNodeProperties(String nodeId)
+            throws IOException
+    {
+        Path tempNodeProperties = Files.createTempFile("node", ".properties");
+
+        String nodeProperties = "node.environment=testing\n" +
+                "node.location=testing-location\n" +
+                "node.id=" + nodeId + "\n";
+        Files.write(tempNodeProperties, nodeProperties.getBytes(), StandardOpenOption.WRITE);
+        return tempNodeProperties;
+    }
+
+    public static MaterializedResult toMaterializedResult(String csvData)
+    {
+        try {
+            List<MaterializedRow> rows = new ArrayList<>();
+            List<Type> columnTypes = new ArrayList<>();
+
+            // Parse CSV data using commons-csv
```

Comment:
We already have dependencies elsewhere on `open-csv`, so let's use that instead.